packages/lesswrong/components/form-components/ImageUpload.tsx

```diff
@@ -165,7 +165,7 @@ const RemoveButton: FC<{
     <Button
       title="Remove"
       onClick={removeImage}
-      className={mainClass}
+      className={classNames("image-remove-button", mainClass)}
```

Comment:
This moves the "Remove" banner image button on mobile up to the top of the page, near the "Change image" button, so that it's actually accessible